lib/iris/tests/unit/experimental/representation/test_CubeRepresentation.py

```diff
@@ -354,14 +354,6 @@ def test_not_included(self):
         for heading in not_included:
             self.assertNotIn(heading, self.result)
 
-    def test_handle_newline(self):
```

Comment:
NOTE: we simply don't output embedded newlines in the "new world" : we quote the string instead.
So this is no longer needed